### ISSUE TITLE
Request failed with status code 500 - expo build:android -t apk

### ISSUE THREAD

**Issue by musthafa-pa** [expo/expo#7292]
Expo diagnostics:
expo diagnostics

  Expo CLI 3.13.1 environment info:
    System:
      OS: Linux 5.3 Ubuntu 19.10 (Eoan Ermine)
      Shell: 5.0.3 - /bin/bash
    Binaries:
      Node: 12.14.1 - /usr/local/bin/node
      Yarn: 1.21.1 - /usr/bin/yarn
      npm: 6.13.4 - /usr/local/bin/npm
    npmPackages:
      expo: ~36.0.0 => 36.0.2 
      react: ~16.9.0 => 16.9.0 
      react-native: https://github.com/expo/react-native/archive/sdk-36.0.0.tar.gz => 0.61.4 
      react-navigation: ^4.2.2 => 4.2.2 
    npmGlobalPackages:
      expo-cli: 3.13.1
Error:

Unable to find an existing Expo CLI instance for this directory, starting a new one...
Starting Metro Bundler on port 19001.
Tunnel ready.
Publishing to channel 'default'...
Building iOS bundle
Building Android bundle
Building JavaScript bundle [===================================================================] 100%Finished building JavaScript bundle in 1198ms.
Analyzing assets
Building JavaScript bundle [===================================================================] 100%Finished building JavaScript bundle in 973ms.
Finished building JavaScript bundle in 905ms.
Finished building JavaScript bundle in 861ms.
Uploading assets
No assets changed, skipped.
Processing asset bundle patterns:
- /home/mustafa/andIos/**/*
Uploading JavaScript bundles
› Closing Expo server
› Stopping Metro bundler
Request failed with status code 500
Set EXPO_DEBUG=true in your env to view the stack trace

**Comment by expjess** [comment]
Hey @musthafa-pa - can you lmk whether you're still seeing this?

**Comment by musthafa-pa** [comment]
Yes. I tried to build the app using all the commands available. But not working
